features/Hotels/hotel.feature

```diff
@@ -0,0 +1,96 @@
+Feature: Project 1
+
+    @report1
+    Scenario: TC-18 Verify user can update number of guests on Homepage
+        Given I am on hotels HomePage
+        When I click on Travelers
+        Then I select adults as 6
+        Then I select children as 3
+        Then I select first child age as 4
+        Then I select second child age under 1
+        Then I select third child age 7
```

Comment:
Pls follow Gherkin rules for all test cases.
Action step => When
Verification step => Then
Any repeated When/Then => And